spec/cc/engine/analyzers/php/main_spec.rb

```diff
@@ -2,12 +2,17 @@
 require 'cc/engine/analyzers/php/main'
 require 'cc/engine/analyzers/reporter'
 require 'cc/engine/analyzers/engine_config'
+require 'cc/engine/analyzers/file_list'
 require 'flay'
 require 'tmpdir'
 
 module CC::Engine::Analyzers::Php
   describe Main do
-    before { @code = Dir.mktmpdir }
+    before do
+      @code = Dir.mktmpdir
+      Dir.chdir(@code)
```

Comment:
That's fair, on OS X they're deleted about every 3 days at least. I do agree that it's probably a good practice though. I'll add that in.